vite.config.ts

```diff
@@ -0,0 +1,65 @@
+import path from 'path'
+import { BuildOptions, defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue2'
+import basicSsl from '@vitejs/plugin-basic-ssl'
+import { loadEnv } from 'vite'
+
+const lib: BuildOptions['lib'] = {
+  entry: path.resolve(__dirname, './src/main.lib.ts'),
+  name: 'OCFilePicker',
+  fileName: 'file-picker',
+  formats: ['es', 'umd']
+}
+
+const wc: BuildOptions['lib'] = {
+  entry: path.resolve(__dirname, './src/main-wc.ts'),
+  name: 'OCFilePicker',
+  fileName: 'file-picker',
+  formats: ['es', 'umd', 'iife']
+}
+
+export default defineConfig(({ mode }) => {
+  const env = loadEnv(mode, process.cwd(), '')
+
+  return {
+    plugins: [basicSsl(), vue()],
+    resolve: {
+      alias: [
+        {
+          find: '@',
+          replacement: path.resolve(__dirname, './src')
+        },
+        {
+          find: 'stream',
+          replacement: `stream-browserify`
+        }
+      ]
+    },
+    preview: {
+      https: true,
+      port: 8080,
```

Comment:
The default dev setup in Web already uses this port for oC 10 - mind to switch to another one?

(ftr 8081 is for vite:oc10, 9200 for ocis, 9201 for vite:ocis)